src/Framework/test/SharedFxTests.cs

```diff
@@ -26,6 +26,36 @@ public SharedFxTests(ITestOutputHelper output)
             _sharedFxRoot = Path.Combine(TestData.GetTestDataValue("SharedFrameworkLayoutRoot"), "shared", "Microsoft.AspNetCore.App", TestData.GetTestDataValue("RuntimePackageVersion"));
         }
 
+        [Fact]
+        public void SharedFrameworkContainsListedAssemblies()
+        {
+            var actualAssemblies = Directory.GetFiles(_sharedFxRoot, "*.dll")
+                .Select(Path.GetFileNameWithoutExtension)
+                .ToHashSet();
+            var listedSharedFxAssemblies = TestData.ListedSharedFxAssemblies;
+
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                listedSharedFxAssemblies.Remove("aspnetcorev2_inprocess");
```

Comment:
Hmm now I'm wondering why this test didn't fail in master without adding this.

Reply:
Ah it's probably because it's testing the targeting pack that was built on Windows, given that we run the test on Helix. It should be benign then since we only ship the package built on Windows anyway.

Reply:
Good points. I'll address 1. As for 2, the reason is that we are running the tests on Helix and we use the test payload that was built on Windows even if we run the tests on non-Windows. @HaoK do we have infrastructure to build test payloads on non-Windows OS's. I don't remember having anything like that so maybe we should be running the tests on AzDO until we can do that?

Reply:
We don't have infrastructure to build on non windows yet, so maybe this test should have SkipTests explicitly set to false so it runs on azdo as well, since the behavior is different.

Reply:
These seem like a good candidate for tests that we just never run on helix too, if they are testing installer/platform specific things?

Reply:
I'll make sure the tests run when I merge this forward to master.